src/app/core/guards/current-user.ts

```diff
@@ -76,32 +93,38 @@ currentSubscription: Subscription=null;
 
 
    }
-
-   public async defaultBusiness() {
-    await this.database.get(PouchConfig.Tables.business).then(res=> {
-
-        const currentBusiness: Business=res.businesses.find(business=>business.active);
```

Comment:
How do we know if this currentBusiness is not null? I would suggest waiting for it to come or check for nullity here.